index.d.ts

```diff
@@ -0,0 +1,12 @@
+type Loggable = string | number | {};
+
+interface ILog {
+    info(message: Loggable): void;
+    debug(message: Loggable): void;
+    warn(message: Loggable): void;
+    error(message: Loggable): void;
+    trace(message: Loggable): void;
+}
+
+declare function logger(source: string): ILog;
+export = logger;
```

Comment:
You need it otherwise you get the error `A 'declare' modifier is required for a top level declaration in a .d.ts file.`